src/Raven.Studio/typescript/viewmodels/resources/createDatabase.ts

```diff
@@ -242,20 +245,106 @@ class createDatabase extends dialogViewModelBase {
             return "unchecked";
         });
 
-
-        this.databaseLocationShowing = ko.pureComputed(() => {
-            return this.databaseLocationCalculated();
-        });
-
         this.databaseModel.path.dataPath.throttle(300).subscribe((newPathValue) => {
             if (this.databaseModel.path.dataPath.isValid()) {
-                new getDatabaseLocationCommand(this.databaseModel.name(), newPathValue)
-                    .execute()
-                    .done((fullPath: string) => this.databaseLocationCalculated(fullPath))
+                this.updateDatabaseLocationInfo(this.databaseModel.name(), newPathValue);
+                this.updateFolderPathOptions(newPathValue);
             } else {
-                this.databaseLocationCalculated("Invalid path");
+                this.databaseLocationInfo([]);
+                this.spinners.databaseLocationInfoLoading(false);
             }
         });
+
+        this.databaseLocationInfoToDisplay = ko.pureComputed(() => {
+            const databaseLocationInfo = this.databaseLocationInfo();
+            const selectedNodes = this.databaseModel.replication.nodes().map(x => x.tag());
+            const replicationFactor = this.databaseModel.replication.replicationFactor();
+            if (this.databaseModel.replication.manualMode()) {
+                if (selectedNodes.length === 0) {
+                    return [];
+                }
+            }
```

Comment:
please remove line break after } and before else ;-)

Reply:
done